servicenow_connector.py

```diff
@@ -1498,14 +1564,15 @@ def _run_query(self, param):
 
         lookup_table = param[SERVICENOW_JSON_QUERY_TABLE]
         query = param[SERVICENOW_JSON_QUERY]
-        endpoint = SERVICENOW_BASE_QUERY_URI + lookup_table + "?" + query
-        limit = self._validate_integers(action_result, param.get(SERVICENOW_JSON_MAX_RESULTS, SERVICENOW_DEFAULT_MAX_LIMIT), SERVICENOW_JSON_MAX_RESULTS)
+        endpoint = '{}{}?{}'.format(SERVICENOW_BASE_QUERY_URI, lookup_table, query)
+        limit = self._validate_integers(action_result, param.get(SERVICENOW_JSON_MAX_RESULTS,
+                    SERVICENOW_DEFAULT_MAX_LIMIT), SERVICENOW_JSON_MAX_RESULTS)
```

Comment:
suggestion: Do you think we can fit all the params in the 2nd line?
```
limit = self._validate_integers(
     action_result, param.get(SERVICENOW_JSON_MAX_RESULTS, SERVICENOW_DEFAULT_MAX_LIMIT), SERVICENOW_JSON_MAX_RESULTS)
```

Reply:
Yes I have made the suggested changes.